app/views/admin/users/edit.html.erb

```diff
@@ -1,6 +1,10 @@
 <h2><%= @user['id'] %></h2>
 
-<%= form_for :user, url: environment_admin_user_path(environment: params[:environment], id: @user['id']), method: :patch do |f| %>
+<%= form_for :user,
+             url: environment_admin_user_path(environment: params[:environment],
+             id: @user['id']),
```

Comment:
is `@user` a hash? are we not able to use dot notation here `@user.id`?

Reply:
@user is a hash that is converted to a json response from the api.